app/src/main/java/org/p2p/wallet/common/ui/bottomsheet/BaseDoneBottomSheet.kt

```diff
@@ -23,16 +24,25 @@ abstract class BaseDoneBottomSheet : BottomSheetDialogFragment() {
     private val resultKey: String by args(ARG_RESULT_KEY)
     private val requestKey: String by args(ARG_REQUEST_KEY)
 
-    lateinit var binding: DialogBaseDoneBinding
+    lateinit var baseDialogBinding: DialogBaseDoneBinding
 
+    @CallSuper
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        binding = DialogBaseDoneBinding.inflate(inflater, container, false)
-        return binding.root
+        baseDialogBinding = DialogBaseDoneBinding.inflate(inflater, container, false)
+        val innerView = onCreateInnerView(inflater, container, savedInstanceState)
+        baseDialogBinding.viewInner.addView(innerView)
+        return baseDialogBinding.root
     }
 
+    abstract fun onCreateInnerView(
```

Comment:
oh yes, the inheritance problem
i wanted to write about it, about the problem of resolving multiple bindings - BaseDoneBottomSheet's and its children
but i guess you figured it out